Sources/FoundationEssentials/Data/Data+Writing.swift

```diff
@@ -361,6 +361,9 @@ private func writeToFileAux(path inPath: PathOrURL, buffer: UnsafeRawBufferPoint
         // We're done now
         guard let auxPath else { return }
 
+        _close(fd)
```

Comment:
I'm curious what the difference between windows and non-windows is here, since the setup is the same for both right. Is this just a subtle behavioral difference between `rename` on POSIX systems and `MoveFileExW` on windows? Or is there another difference in the code that leads to this?

Reply:
Yes, it's the use of the windows APIs and the requirement to close the handle before the move.